tests/dags/maintenance/test_pr_review_reminders.py

```diff
@@ -166,15 +179,82 @@ def test_does_not_reping_past_due(github, urgency):
     assert not_due_pull["number"] not in github["posted_comments"]
 
 
+@parametrize_urgency
+def test_does_reping_past_due_if_reminder_is_outdated(github, urgency):
+    past_due_pull = make_pull(urgency, past_due=True)
+    past_due_pull["requested_reviewers"] = [
+        make_requested_reviewer(f"reviewer-due-{i}") for i in range(2)
+    ]
+    not_due_pull = make_pull(urgency, past_due=False)
+    not_due_pull["requested_reviewers"] = [
+        make_requested_reviewer(f"reviewer-not-due-{i}") for i in range(2)
+    ]
+    github["pulls"] += [past_due_pull, not_due_pull]
+    github["pull_comments"][past_due_pull["number"]].append(
+        make_outdated_pr_comment(past_due_pull)
+    )
+
+    post_reminders("not_set", dry_run=False)
+
+    assert past_due_pull["number"] in github["posted_comments"]
+    assert not_due_pull["number"] not in github["posted_comments"]
+
+
+UNAPPROVED_REVIEW_STATES = ("CHANGES_REQUESTED", "COMMENTED")
+
+
+@parametrize_urgency
 @pytest.mark.parametrize(
-    "urgency",
-    (
-        Urgency.CRITICAL,
-        Urgency.HIGH,
-        Urgency.MEDIUM,
-        Urgency.LOW,
-    ),
+    "first_review_state",
+    # None represents not having a review so we can test the
+    # one review state.
+    (None,) + UNAPPROVED_REVIEW_STATES,
```

Comment:
Nit: This threw me off a little initially with the single item tuple declaration and tuple concatenation. Would you be alright changing this to a splat? (same for the second parametrization)
```suggestion
    (None, *UNAPPROVED_REVIEW_STATES),
```

Reply:
:sweat_smile: Interesting! @dhruvkb strongly prefers the `|` operator on dicts, I prefer the splat there. Here, I prefer the `+` on iterables, but you prefer the splat! To me it makes little difference, though I do think the `+` is a lot more readable.

Reply:
Alternatively, I could make them lists if that would be clearer, though I do think they should be tuples.

Reply:
Ha! 😂 🚲 🏠 

Either is fine, really. I do believe that pytest params can be lists, perhaps that's the way to go 🙂

Reply:
They can be, but to me lists imply mutability and the constant is not mutable.